components/up-job.libsonnet

```diff
@@ -0,0 +1,79 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
+
+{
+  local up = self,
+
+  config:: {
+    name: error 'must provide name',
```

Comment:
What's this? :scream:  Was not aware of this feature.

Reply:
one of my fav features! that and assert for testing are the most underused and underappreciated features